tools/code_format/format_python_tools.py

```diff
@@ -14,7 +14,6 @@ def collect_files():
     Returns: A collection of python files in the tools directory excluding
        any directories in the EXCLUDE_LIST constant.
     """
-    # TODO: Add ability to collect a specific file or files.
```

Comment:
this is resolved at least for the case of a single file - this method is then avoided